examples/with-supertokens/app/components/getServerSideSession.tsx

```diff
@@ -0,0 +1,90 @@
+import { cookies, headers } from 'next/headers';
+import { NextRequest, NextResponse } from 'next/server';
+import Session, { SessionContainer } from 'supertokens-node/recipe/session';
+import { ensureSuperTokensInit } from '../../config/backendConfig';
+
+ensureSuperTokensInit();
+
+export async function getServerSideSession(req?: NextRequest): Promise<{
+  session: SessionContainer | undefined;
+  hasToken: boolean;
+  resp?: Response;
```

Comment:
do we need to return the response object?

Reply:
I think it's more convenient.

Reply:
Oh, but i don't see it being used anywhere.